src/app/test/test.component.html

```diff
@@ -5,16 +5,16 @@
   <div class="col">
     <div class="row">
       <app-button [icon]="'fa fa-play'" [title]="'Run All'" (click)="runSelected()"></app-button>
-      <app-button [icon]="'fa fa-refresh'" [title]="'ReloadTestReports'" (click)="loadData(null)"></app-button>
+      <app-button data-cy-test-table="refresh" [icon]="'fa fa-refresh'" [title]="'ReloadTestReports'" (click)="loadData(null)"></app-button>
```

Comment:
Deze refresh button heeft niks te maken met de table
```suggestion
      <app-button data-cy-test-button="refresh" [icon]="'fa fa-refresh'" [title]="'ReloadTestReports'" (click)="loadData(null)"></app-button>
```